ui/src/chatbot.tsx

```diff
@@ -32,44 +32,39 @@ const
     },
     msgContainer: {
       position: 'absolute',
-      top: 15,
+      top: 0,
       left: 0,
       right: 0,
-      bottom: 62, // Height of input box + padding.
+      bottom: 0,
       overflowY: 'auto',
-      padding: padding(0, 15),
+      padding: padding(0, 0),
+    },
+    msgWrapper: {
+      '&:first-child': { marginTop: important(px(0)) },
+      '&:last-child': { marginBottom: important(px(100)) },
+      display: 'flex',
+      justifyContent: 'center',
     },
     msg: {
+      width: '65ch',
+    },
+    userMsg: {
+      backgroundColor: cssVar('$neutralLighter'),
```

Comment:
Let's swap the current background colorss for bot and user - user should have no background (card color) and bot neutralLighter as his responses are more important (should be kinda highlighted).